01-backup.md

```diff
@@ -54,6 +54,23 @@ we're telling Git:
 
 The four commands above only need to be run once:
 the flag `--global` tells Git to use the settings for every project on this machine.
+The configuration options get stored in your home directory in a text file called `.gitconfig`.
+
+~~~ {.bash}
+$ cat ~/.gitconfig
+~~~
+~~~ {.output}
+[user]
+	name = Vlad Dracula
+	email = vlad@tran.sylvan.ia
+[color]
+	ui = true
+[core]
+	editor = nano
+~~~
+
+This file can be edited directly.
```

Comment:
I would like this in the lesson, but again, I don't think it should go 
in `01-backup.md`: our learners are already overwhelmed with new 
information, and this can (and therefore should) be delayed.